lib/message-routing/message-router.js

```diff
@@ -36,32 +37,34 @@ class MessageRouter {
       return;
     }
 
-    const bannedPhrase = this.spamDetection.checkAgainstBannedPhrases(messageContent);
-
-    if (bannedPhrase !== false) {
-      if (bannedPhrase.type === 'mute') {
-        this.punishmentStream.write(
-          makeMute(
-            user,
-            bannedPhrase.duration,
-            `${user} muted for using banned phrase(${bannedPhrase.text}).`,
-            true,
-          ),
-        );
-      } else if (bannedPhrase.type === 'ban') {
-        this.punishmentStream.write(
-          makeBan(
-            user,
-            bannedPhrase.duration,
-            null,
-            null,
-            `${user} banned for using banned phrase(${bannedPhrase.text}).`,
-            true,
-          ),
-        );
+    if (this.chatConnectedTo === 'twitch') {
```

Comment:
This means chat-bot shouldn't bother checking incoming messages on the assumption that it will never encounter an message containing a banned phrase outside of Twitch chat, correct?

Reply:
Yeah, it skips the built-in banned phrase check in d.gg chat.

Reply:
I was considering whether it's better to not load banned phrases from the database at all, but don't think it's a big deal.